src/oc-id/Makefile

```diff
@@ -1,5 +1,9 @@
+# TODO - looks like we can get built in support for rvm
```

Comment:
I looked into this but couldn't figure out how.  Their built-in ruby installer only works if you set the `language` to ruby.  If we set the language to ruby then the build-in erlang installer doesn't work.  It looked easier at the time to manage ruby on our own than erlang.